### PR TITLE
Inverter v2

### PR DESCRIPTION
**Pull request by NormannK**
- rewritten/translated the inverter class
- add tests
- same logic as #164 where I did show that the new tests are also working on the old class

Todo:
- [x] rename the class itself
- [x] translate in devices.py

### CODE REVIEW

**Comment by NormannK** [comment]
done, ready for review and merging

File: src/akkudoktoreos/optimization/genetic.py [hunk 1]
```diff
@@ -488,10 +488,10 @@ def optimierung_ems(
         )
 
         # Initialize the inverter and energy management system
-        wr = Wechselrichter(parameters.wechselrichter, akku)
+        wr = Inverter(parameters.inverter, akku)
```
Comment (Lasall) [inline src/akkudoktoreos/optimization/genetic.py:491]: variable name could be translated

Comment (NormannK) [reply src/akkudoktoreos/optimization/genetic.py:491]: yes, but this is the pr for the inverter file. I cant translate all parts of all files. that would be an endless rabbit hole. 
imo battery must come next.

File: tests/testdata/optimize_input_1.json [hunk 2]
```diff
@@ -1,58 +1,60 @@
 {
-    "ems": {
```
Comment (Lasall) [inline tests/testdata/optimize_input_1.json:2]: could you keep indentation=4 so it's easier to see what really changed

File: src/akkudoktoreos/optimization/genetic.py (hunk 1 above)
Comment (NormannK) [reply src/akkudoktoreos/optimization/genetic.py]: done anyway ;-)

File: tests/testdata/optimize_input_1.json (hunk 2 above)
Comment (NormannK) [reply tests/testdata/optimize_input_1.json:2]: hmm visual studio did that automatically. any idea how to prevent it?

**Review by Lasall — APPROVED** [review]

**Pull request merged**